src/components/profile/currentUserContext.js

```diff
@@ -35,6 +50,7 @@ export const CurrentUserProvider = ({ children }) => {
           return Promise.reject(err);
         })
         .finally(() => {
+          setLastFetchCurrentUser(Date.now());
```

Comment:
I'd actually set this from the moment the call is made. We don't care whether the call succeeds or fails, we just want to throttle all calls

Reply:
Makes sense. I'll make this change in the other api calls.

Reply:
fixed